src/BaselineOfFreeType/BaselineOfFreeType.class.st

```diff
@@ -0,0 +1,30 @@
+Class {
+	#name : #BaselineOfFreeType,
+	#superclass : #BaselineOf,
+	#category : #BaselineOfFreeType
+}
+
+{ #category : #baselines }
+BaselineOfFreeType >> baseline: spec [
+	<baseline>
+	
+	spec for: #common do: [ 
+		spec package: 'FreeType'.
+		spec package: 'FreeType-Morphic'.
+		spec package: 'FreeType-Tests'.
```

Comment:
Dependencies among packages should be declared, no?

Like for example:
```Smalltalk
spec package: 'FreeType-Tests' with: [ spec requires: #('FreeType' 'FreeType-Morphic') ]
```

Reply:
I will merge the current state for now...